android/sdl_android/src/main/java/com/smartdevicelink/transport/utl/SdlDeviceListener.java

```diff
@@ -197,7 +197,7 @@ public static void setSDLConnectedStatus(Context context, String address, boolea
                 }
                 SharedPreferences.Editor editor = preferences.edit();
                 editor.putBoolean(address, hasSDLConnected);
-                editor.commit();
+                editor.apply();
```

Comment:
We specifically want to wait for this commit to happen; apply will return immediately.